iocore/net/SSLNetVConnection.cc

```diff
@@ -1812,6 +1813,36 @@ SSLNetVConnection::populate(Connection &con, Continuation *c, void *arg)
   return EVENT_DONE;
 }
 
+void
+SSLNetVConnection::increment_ssl_version_metric(const char *version) const
+{
+  if (version) {
+    // openSSL guarantees the case of the protocol string.
+    if (version[0] == 'T' && version[1] == 'L' && version[2] == 'S' && version[3] == 'v' && version[4] == '1') {
```

Comment:
strncmp for better readability.

Reply:
Because this is SSLNetVConnection, we should be able to use `SSL_version` instead of `SSL_get_version` (via `getSSLProtocol`). Then we can replace these `if` statements with a simple `switch` statement.

Reply:
+1 - I like this suggestion better.

Reply:
> strncmp for better readability.
Yeah, I was debating the use of strncmp(), but, followed what is written for protocol_contains() to keep it consistent. In any case, @maskit 's suggestion to use SSL_version() feels a lot better than having to much around with strings, so, I'm going to just flip to use that instead!